CommandLine/libEGM/action.cpp

```diff
@@ -75,6 +75,39 @@ std::string Actions2Code(const std::vector< buffers::resources::Action >& action
   for (const auto &action : actions) {
     const auto &args = action.arguments();
 
+    std::string applies_to = action.who_name();
+    if (action.use_apply_to() && applies_to != "self") {
+      const auto &ref_as_int = [&action](const std::string &applies_to) -> std::string {
+        if (applies_to == "self") {
+          return "-1";

```

Comment:
Idont think self / other need to be converted to ints

Reply:
I mentioned that because the enum containing these keywords itself maps them to the ints.